react-ui/src/components/Layouts/InputWorkerTimes/CheckboxApplication.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { render } from 'react-dom'; 
+import './CheckboxApplication.scss';
+ import { Container, Row, Col } from 'react-grid-system';
+import { Redirect, Route, withRouter, Link, Router } from "react-router-dom"; 
+import moment from 'moment'
+
+class Check extends React.Component { // this is a class component for the checkbox feature of selecting recurring days
+  constructor() {
+    super();
+    this.state = {
+      Days: [
+        { id: 1, value: "Monday" },
+        { id: 2, value: "Tuesday" },
+        { id: 3, value: "Wednesday" },
+        { id: 4, value: "Thursday" },
+        { id: 5, value: "Friday" },
+        { id: 6, value: "Saturday" },
+        { id: 7, value: "Sunday" }
+      ],
+      checkedDays: new Map()
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    var isChecked = event.target.checked;
+    var item = event.target.value;
+    this.setState(prevState => ({ checkedDays: prevState.checkedDays.set(item, isChecked) }));
+  }
+
```

Comment:
I might add a small explanation or "instructions" as to what the date means as I didn't understand at first why I need to to select days of the week AND a date on the calendar. Only after using the application a couple times and seeing the output in the console did I understand what the select date meant

Reply:
Hi Bradley,

I have added a header outlining the basic instructions.
<img width="956" alt="Capture" src="https://user-images.githubusercontent.com/32501521/98073863-42578a80-1e37-11eb-8bf7-5f4a2a6487a0.PNG">


Hopefully, that eliminates the lack of insight for a first time user.

Reply:
Perfect! Thank you Kyle, It looks great!